sql/database.sql

```diff
@@ -1806,7 +1806,27 @@ CREATE TABLE  `facility_user_ids` (
   KEY `uuid` (`uuid`)
 ) ENGINE=InnoDB  AUTO_INCREMENT=1;
 
--- --------------------------------------------------------
+-----------------------------------------------------------
+
+--
+-- Table structure for table `fee_schedule`
+--
+
+DROP TABLE IF EXISTS `fee_schedule`;
+CREATE TABLE `fee_schedule` (
+    `id` BIGINT NOT NULL AUTO_INCREMENT,
+    `insurance_company_id` SMALLINT(4) DEFAULT NULL,
+    `plan` VARCHAR(20),
```

Comment:
Suggest plan and modifier default to null. Will also mean while loading, if input is blank the defaults are respected.

Reply:
the trick will be to make sure the values are set to an empty string when loaded so the unique key will be enabled :)